front/app/containers/Admin/projects/project/general/components/GeographicAreaInputs.tsx

```diff
@@ -130,15 +170,29 @@ const GeographicAreaInputs = ({
           value="none"
           name="areas"
           id="areas-none"
-          label={<FormattedMessage {...messages.areasNoneLabel} />}
+          label={
+            <LabelHeaderDescription
+              header={<FormattedMessage {...messages.areasNoneLabel} />}
+              description={
+                <FormattedMessage {...messages.areasNoneLabelDescription} />
+              }
+            />
+          }
         />
         <Radio
           onChange={handleAreaTypeOnChange}
           currentValue={areaType}
           value="all"
           name="areas"
           id="areas-all"
-          label={<FormattedMessage {...messages.areasAllLabel} />}
+          label={
+            <LabelHeaderDescription
+              header={<FormattedMessage {...messages.areasAllLabel} />}
+              description={
+                <FormattedMessage {...messages.areasAllLabelDescription} />
+              }
+            />
+          }
         />
```

Comment:
Could you add margin-bottom to each Radio? Similar to what we have with the participation method (the screenshot in the ticket).

Reply:
```
export const ParticipationMethodRadio = styled(Radio)`
  margin-bottom: 25px;
`;
```